oneflow/core/eager/lazy_job_instruction_type.cpp

```diff
@@ -93,11 +93,12 @@ class LaunchLazyJobInstructionType final : public InstructionType {  // NOLINT
     const auto& cur_nn_graph = GetCurNNGraph(instruction);
     auto* device_ctx = GetLazyJobDeviceCtx(instruction);
 
+    static thread_local int64_t run_id = 0;
```

Comment:
thread_local 变量肯定是 static 的

Reply:
https://en.cppreference.com/w/cpp/language/storage_duration

It can be combined with static or extern to specify internal or external linkage (except for static data members which always have external linkage), respectively, but that additional static doesn't affect the storage duration.

这里可以加 static，但影响的是 linkage 不是 storage duration